src/shared/api.json

```diff
@@ -50,6 +50,7 @@
   "URL_POLICY_VIOLATION": "api/v1/violation",
   "URL_POLICY_VIOLATION_ANALYSIS": "api/v1/violation/analysis",
   "URL_LICENSE_GROUP": "api/v1/licenseGroup",
+  "URL_ACL_ENABLED": "api/v1/acl/enabled",
```

Comment:
I think this one can be removed as well now.

Reply:
😢

Reply:
removed, good find.